get.md

```diff
@@ -7,13 +7,25 @@ You have a handful of ways to try out LXDE.
 
 #### But I am totally new to those Linux things
 
-We encourage you to try out Lubuntu, which is based on Ubuntu Linux, and made
-by our community members.
+We encourage you to try out Fedora LXDE
 
 <ul class="actions">
-  <li><a href="http://lubuntu.me" class="button special">Get Lubuntu</a></li>
+  <li><a href="https://spins.fedoraproject.org/lxde/" class="button special">Get Fedora LXDE</a></li>
 </ul>
 
+
+#### I'm not new, but I want to install something that comes with LXDE
+
+Many distributions offer LXDE out of the box, such as LXLE and Debian.
```

Comment:
openSUSE as well.

Reply:
There isn't a way to install it from YaST sadly

Reply:
Of course there is. Under the patterns section.
And from cli via `zypper in -t patern lxde`.
It can also be selected as the only DE to install when you use the netinst iso.